build.sbt

```diff
@@ -209,3 +209,13 @@ def monixConnector(
     .configs(IntegrationTest, IT)
 
 //todo add release settings
+
+lazy val gcs = (project in file("gcs"))
```

Comment:
you can shorten the creation of the module by using the `monixConnector()` method from the same build.sbt

Reply:
Will update accordingly :-)